frontend/src/lib/api/sanity.ts

```diff
@@ -1,16 +1,17 @@
-import sanityClient from '@sanity/client';
+import { createClient } from 'next-sanity';
 import imageUrlBuilder from '@sanity/image-url';
+import type { SanityImageSource } from '@sanity/image-url/lib/types/types';
 
-const SanityAPI = sanityClient({
+const SanityAPI = createClient({
     projectId: 'pgq2pd26',
     dataset: process.env.SANITY_DATASET ?? 'production',
     apiVersion: '2021-04-10',
-    useCdn: true,
+    useCdn: typeof document !== 'undefined',
```

Comment:
Denne vil vel være `false` i `getServerSideProps`? Skjønner tankegangen til Sanity sitt eksempel, men `getServerSideProps` er jo 'on demand', og da er CDN bedre.

Reply:
Jeg tror den vil være `false` på stedene vi bruker den. Den anbefaler å bruke CDN på front-end og ikke CDN på back-end. Sånn jeg forsto det. [Link](https://www.sanity.io/docs/api-cdn).

Usikker på som er best å velge her i Next-koden, men jeg er veldig sikker på at clienten i backenden ikke burde bruke CDN.

Reply:
@boaanes Hva mener du?

Reply:
Ja er helt enig med at på alle steder utenom `getServerSideProps` burde den være false, siden man trenger ikke hastighet fra CDN når sider blir generert ved build (som `getStaticProps` osv...). Men i dette tilfelle vil CDN ikke brukes for å laste inn f.eks. `/event/[slug]`, som kan være dumt siden det vil gå tregere.

Reply:
Det er sant. Man kan jo lage to clients?

Reply:
Yes det funker

Reply:
👍🏻 Fikser ila. dagen.

Reply:
@bakseter Jeg tror ikke vi gidder to clients. Det har funket med bare CDN så langt. Om det ikke har vært et problem til nå, så er det ikke noe vits å "fikse" heller.